### PR TITLE
Docs: improve README.md file

### PR DESCRIPTION
**Pull request by poorvika11**
## Description

Readme has been updated with all required things.Please Review it and merge it.

## Screenshots

a
b
c

### CODE REVIEW

File: Readme.md
Comment (priyankarpal) [inline Readme.md:1]: rename to `README.md`

File: p1.png
Comment (priyankarpal) [inline p1.png:1]: don't add an image to the root folder use this method to add an image.

paste the link 


https://github.com/priyankarpal/ProjectsHut/assets/88102392/204a7cf5-c8ed-49bc-825f-4f87cf792626

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+## ✨ How to use it? 
+
+### To install and run Projectshut locally, follow these steps:
+
+- Clone the repository: `git clone https://github.com/yourusername/projectshut.git`
+- Navigate to the project `directory: cd projectshut`
+- Install dependencies: `npm install`
+- Set up the environment variables by creating a `.env` file based on the provided `.env.example` file.
+- Start the application: `npm start`
+- Access the application at `http://localhost:3000`
```
Comment (priyankarpal) [inline Readme.md:60]: we already have the `contributing.md` file so here we need to avoid it.

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+- **Multiple Language Access**: The website is available for multiple languages including 한국인, Española,தமிழ்,German,Italiano,Japanese,French,हिंदी and Tagalog.
+
+
+## 💻 Tech Stack 
+
+  ![reactjs](https://img.shields.io/badge/React-20232A?style=for-the-badge&logo=react&logoColor=61DAFB)&nbsp;
+  ![nodejs](https://img.shields.io/badge/Node.js-43853D?style=for-the-badge&logo=node.js&logoColor=white)&nbsp;
+  ![expressjs](https://img.shields.io/badge/Express.js-000000?style=for-the-badge&logo=express&logoColor=white)&nbsp;
+  ![PNPM](https://img.shields.io/badge/pnpm-%234a4a4a.svg?style=for-the-badge&logo=pnpm&logoColor=f69220)&nbsp;
```
Comment (priyankarpal) [inline Readme.md:47]: remove extra tags

File: Readme.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+- **Collaboration Opportunities**: Connect with like-minded individuals, collaborate on projects, and exchange ideas
+
+- **Project Publication**: Easily publish your projects on Projectshut, providing a platform for others to discover and explore your work.
+
+- **Project Categorization**: Organize your projects into categories and tags for easy search and discovery. 
+
+- **Notifications**: Stay up-to-date with notifications about project updates, comments, and new collaborations.
+
+- **Multiple Language Access**: The website is available for multiple languages including 한국인, Española,தமிழ்,German,Italiano,Japanese,French,हिंदी and Tagalog.
```
Comment (priyankarpal) [inline Readme.md:39]: please add languages to an unorderly & also, add the link of their path

File: Readme.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
+
+<h1 align=center> ProjectsHut </h1>
+
+<p align=center>
+Welcome to the ProjectsHut GitHub repository! ProjectsHut is an open-source web application that empowers users to share and promote their projects with ease. Our platform provides a user-friendly interface for publishing projects and creating personalized user profiles. The primary goal of ProjectsHut is to cultivate a collaborative community of creative individuals, where ideas flourish and innovation thrives. Whether you're a developer, designer, or an enthusiastic project enthusiast, ProjectsHut offers the perfect platform to showcase your work and connect with like-minded individuals.  </p>
```
Comment (priyankarpal) [inline Readme.md:5]: left align text

File: Readme.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  <img src="https://img.shields.io/github/issues/priyankarpal/ProjectsHut-api.svg?style=for-the-badge&logo=appveyor" />
+  <img src="https://img.shields.io/github/issues-closed-raw/priyankarpal/ProjectsHut-api.svg?style=for-the-badge&logo=appveyor" />
+  <br>
+  <img src="https://img.shields.io/github/forks/priyankarpal/ProjectsHut-api.svg?style=for-the-badge&logo=appveyor" />
+  <img src="https://img.shields.io/github/issues-pr/priyankarpal/ProjectsHut-api.svg?style=for-the-badge&logo=appveyor" />
+  <img src="https://img.shields.io/github/issues-pr-closed-raw/priyankarpal/ProjectsHut.svg?style=for-the-badge&logo=appveyor" />
+  <br>
+  <img src="https://img.shields.io/github/stars/priyankarpal/ProjectsHut-api.svg?style=for-the-badge&logo=appveyor" />
+  <img src="https://img.shields.io/github/last-commit/priyankarpal/ProjectsHut-api.svg?style=for-the-badge&logo=appveyor" />
+</div>
```
Comment (priyankarpal) [inline Readme.md:24]: also, remove unwanted badges

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by poorvika11** [comment]
@priyankarpal the changes I wanted were solved through this PR. You can check there. Also I have made changes you wanted

**Comment by priyankarpal** [comment]
> @priyankarpal the changes I wanted were solved through this PR. You can check there. Also I have made the changes you wanted

yeah just add the image(copy the link from readme)
image

**Review by priyankarpal — CHANGES_REQUESTED** [review]
also, rename the PR title to `Docs: improve README.md file`

**Comment by poorvika11** [comment]
@priyankarpal done

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Comment by poorvika11** [comment]
Ohkay sure. I will make sure to apply this from every next commit

**Pull request merged**